Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -19,465 +19,555 @@
+    {{3013, 0x2000}, {"Grim Creeper", Game::SwapForce, Element::Undead, Type::Skylander}},
+    {{3013, 0x2206},
+     {"Grim Creeper (Lightcore)", Game::SwapForce, Element::Undead, Type::Skylander}},
+    {{3013, 0x2603},
+     {"Grim Creeper (Legendary) (Lightcore)", Game::SwapForce, Element::Undead, Type::Skylander}},
+    {{3014, 0x2000}, {"Rip Tide", Game::SwapForce, Element::Water, Type::Skylander}},
+    {{3015, 0x2000}, {"Punk Shock", Game::SwapForce, Element::Water, Type::Skylander}},
+    {{3400, 0x4100}, {"Fiesta", Game::Superchargers, Element::Undead, Type::Skylander}},
+    {{3400, 0x4515}, {"Fiesta (Frightful)", Game::Superchargers, Element::Undead, Type::Skylander}},
+    {{3401, 0x4100}, {"High Volt", Game::Superchargers, Element::Tech, Type::Skylander}},
+    {{3402, 0x4100}, {"Splat", Game::Superchargers, Element::Magic, Type::Skylander}},
+    {{3402, 0x4502}, {"Splat (Power Blue)", Game::Superchargers, Element::Magic, Type::Skylander}},
+    {{3406, 0x4100}, {"Stormblade", Game::Superchargers, Element::Air, Type::Skylander}},
+    {{3406, 0x4502}, {"Stormblade (Dark)", Game::Superchargers, Element::Air, Type::Skylander}},
+    {{3406, 0x4503}, {"Stormblade (Dark)", Game::Superchargers, Element::Air, Type::Skylander}},
+    {{3411, 0x4100}, {"Smash Hit", Game::Superchargers, Element::Earth, Type::Skylander}},
+    {{3411, 0x4502},
+     {"Smash Hit (Steel Plated)", Game::Superchargers, Element::Earth, Type::Skylander}},
+    {{3412, 0x4100}, {"Spitfire", Game::Superchargers, Element::Fire, Type::Skylander}},
+    {{3412, 0x4502}, {"Spitfire (Dark)", Game::Superchargers, Element::Fire, Type::Skylander}},
+    {{3412, 0x450F}, {"Spitfire (Instant)", Game::Superchargers, Element::Fire, Type::Skylander}},
+    {{3413, 0x4100}, {"Jet-Vac (Hurricane)", Game::Superchargers, Element::Air, Type::Skylander}},
+    {{3413, 0x4503},
+     {"Jet-Vac (Legendary Hurricane)", Game::Superchargers, Element::Air, Type::Skylander}},
+    {{3414, 0x4100},
+     {"Trigger Happy (Double Dare)", Game::Superchargers, Element::Tech, Type::Skylander}},
+    {{3414, 0x4502},
+     {"Trigger Happy (Power Blue)", Game::Superchargers, Element::Tech, Type::Skylander}},
+    {{3415, 0x4100},
+     {"Stealth Elf (Super Shot)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3415, 0x4502},
+     {"Stealth Elf (Dark Super Shot)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3415, 0x450F},
+     {"Stealth Elf (Instant)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3416, 0x4100},
+     {"Terrafin (Shark Shooter)", Game::Superchargers, Element::Earth, Type::Skylander}},
+    {{3417, 0x4100},
+     {"Roller Brawl (Bone Bash)", Game::Superchargers, Element::Undead, Type::Skylander}},
+    {{3417, 0x4503},
+     {"Roller Brawl (Legendary Bone Bash)", Game::Superchargers, Element::Undead, Type::Skylander}},
+    {{3420, 0x4100}, {"Big Bubble Pop Fizz", Game::Superchargers, Element::Magic, Type::Skylander}},
```

Comment:
(Big Bubble) maybe?

Reply:
For sure. Don’t know why that one doesn’t use the same naming convention